new-lamassu-admin/src/components/expandable-table/ExpTable.js

```diff
@@ -28,7 +28,7 @@ const ExpRow = ({ id, columns, details, sizes, expanded, className, expandRow, .
 
   const detailsRowClasses = {
     [classes.detailsRow]: true,
-    [classes.hideDetailsRow]: expanded
+    [classes.hideDetailsRow]: !expanded
```

Comment:
Prevent the component from being mounted instead of using a class 

`{expanded && (<Tr....)`